### ISSUE TITLE
TestScan.test__call__ failed

### ISSUE THREAD

**Issue by fajinyuan** [DiamondLightSource/diffcalc#43]
```
[1m[31m____________________________ TestScan.test__Call__ _____________________________[0m

self = <test.diffcalc.gdasupport.minigda.test_command.TestScan object at 0x7f702b236fd0>

[1m    def test__Call__(self):[0m
[1m        scn4 = SingleFieldDummyScannable('scn4')[0m
[1m        scn4.setLevel(4)[0m
[1m        scn5a = SingleFieldDummyScannable('scn5a')[0m
[1m        scn5a.setLevel(5)[0m
[1m        scn5b = SingleFieldDummyScannable('scn5b')[0m
[1m        scn5b.setLevel(5)[0m
[1m        scn6 = SingleFieldDummyScannable('scn6')[0m
[1m        scn6.setLevel(6)[0m
[1m>       self.scan.__call__(scn5a, 1, 3, 1, scn6, 1, scn5b, scn4)[0m

[1m[31mtest/diffcalc/gdasupport/minigda/test_command.py[0m:138: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[1m[31mdiffcalc/gdasupport/minigda/command.py[0m:217: in __call__
[1m    self._performScan(groups, currentRecursionLevel=0)[0m
[1m[31mdiffcalc/gdasupport/minigda/command.py[0m:262: in _performScan
[1m    self._performScan(groups, currentRecursionLevel + 1)[0m
[1m[31mdiffcalc/gdasupport/minigda/command.py[0m:272: in _performScan
[1m    for handler in self.dataHandlers: handler.callWithScanPoint(posDict)[0m
[1m[31mdiffcalc/gdasupport/minigda/command.py[0m:169: in callWithScanPoint
[1m    self.print_first_point(position_dict)[0m
```

**Comment by fajinyuan** [comment]
cause by commit  e05bd61 related to issue #38 - solution is not to remove methods from ScannableBase but override it in sub-class with the same implementations for these methods

**Comment by fajinyuan** [comment]
fixed at commit c8567ba0f55e1ac02ffcd3b8911845d31f86d3e0
